### PR TITLE
Corrected wps installation

### PR DESCRIPTION
**Pull request by garvct**
- Added packages (jasper-devel & libpng-devel)
-Corrected path to WRF installation.

### CODE REVIEW

File: apps/wrf/install_wps_mvapich2.sh [hunk 1]
```diff
@@ -23,7 +26,7 @@ module load gcc-9.2.0
 
 export HDF5=$(spack location -i hdf5^mvapich2)
 export NETCDF=$(spack location -i netcdf-fortran^mvapich2)
-export WRF_DIR=${SHARED_APP}/WRF-${WRF_VERSION}
+export WRF_DIR=${SHARED_APP}/${SKU_TYPE}/wrf-mvapich2/WRF-${WRF_VERSION}
```
Comment (xpillons) [inline apps/wrf/install_wps_mvapich2.sh]: Can't the APP_DIR be reused here ?
Is there any differences between WRF_VERSION and APP_VERSION ?

File: apps/wrf/install_wps_openmpi.sh [hunk 2]
```diff
@@ -23,7 +26,7 @@ module load gcc-9.2.0
 
 export HDF5=$(spack location -i hdf5^openmpi)
 export NETCDF=$(spack location -i netcdf-fortran^openmpi)
-export WRF_DIR=${SHARED_APP}/WRF-${WRF_VERSION}
+export WRF_DIR=${SHARED_APP}/${SKU_TYPE}/wrf-openmp/WRF-${WRF_VERSION}
```
Comment (xpillons) [inline apps/wrf/install_wps_openmpi.sh]: is it wrf_openmpi or wrf-openmpi ?
Same feedbacks as for mvapich2 regarding reusing APP_DIR

File: apps/wrf/install_wps_mvapich2.sh (hunk 1 above)
Comment (garvct) [reply apps/wrf/install_wps_mvapich2.sh]: The WPS installation needs the path to WRF and so APP_DIR cannot be used (it is location of WPS and not WRF).
APP_VERSION is WPS version (4.1), but I also need WRF version (WRF_VERSION) which is different, 4.1.3.

File: apps/wrf/install_wps_openmpi.sh (hunk 2 above)
Comment (garvct) [reply apps/wrf/install_wps_openmpi.sh]: wrf-openmpi is correct (not wrf_openmpi).

**Pull request merged**